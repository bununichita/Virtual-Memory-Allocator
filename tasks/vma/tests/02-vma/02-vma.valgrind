==36834== Memcheck, a memory error detector
==36834== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==36834== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==36834== Command: /home/nicky/Desktop/SDA/Virtual-Memory-Allocator/vma
==36834== Parent PID: 36833
==36834== 
==36834== 
==36834== HEAP SUMMARY:
==36834==     in use at exit: 0 bytes in 0 blocks
==36834==   total heap usage: 16 allocs, 16 frees, 8,514 bytes allocated
==36834== 
==36834== All heap blocks were freed -- no leaks are possible
==36834== 
==36834== For lists of detected and suppressed errors, rerun with: -s
==36834== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
