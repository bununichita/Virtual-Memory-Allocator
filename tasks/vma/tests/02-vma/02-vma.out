Total memory: A bytes
Free memory: 8 bytes
Number of allocated blocks: 2
Number of allocated miniblocks: 2

Block 1 begin
Zone: 1 - 2
Miniblock 1:	1	-	2	| RW-
Block 1 end

Block 2 begin
Zone: 5 - 6
Miniblock 1:	5	-	6	| RW-
Block 2 end
Total memory: A bytes
Free memory: 9 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 1

Block 1 begin
Zone: 5 - 6
Miniblock 1:	5	-	6	| RW-
Block 1 end
